omnibus/files/private-chef-ctl-commands/rotate_credentials.rb

```diff
@@ -85,22 +94,28 @@
 add_command_under_category "rotate-shared-secrets", "credential-rotation", "Rotate the Chef Server shared secrets and all service credentials", 2 do
   ensure_configured!
 
-  begin
-    timestamp = Time.now.iso8601
-    secrets_file = "/etc/opscode/private-chef-secrets.json"
-    secrets_file_backup = File.expand_path(File.join(File.dirname(secrets_file), "#{File.basename(secrets_file)}#{timestamp}"))
-    log("Backing up #{secrets_file} to #{secrets_file_backup}...")
-    FileUtils.cp(secrets_file, secrets_file_backup)
+  secrets_file = "/etc/opscode/private-chef-secrets.json"
```

Comment:
You assign this path to the secrets_file here, but not in the command above. Should probably at least be consistent.

This path actually gets hard coded in a lot of places. I've debated if it's worth asking you to hard code it and then pass it around to the methods, but I'm not sure it's worth it. Another option would be to make secrets_file a method that just returns this path, since it seems to always be the same.